.github/workflows/deploy.yml

```diff
@@ -43,6 +43,9 @@ jobs:
           elif [[ "${{github.base_ref}}" == "dev" || "${{github.ref}}" == "refs/heads/dev" ]]; then
             echo "Setting dev environment"
             echo "environment=dev" >> $GITHUB_OUTPUT
+          elif [[ "${{github.base_ref}}" == "feat/network-statistics-endpoint" || "${{github.ref}}" == "refs/heads/feat/network-statistics-endpoint" ]]; then
```

Comment:
We should undo this before merging

Reply:
Done: 3d732c8b5f5671ca12423c63fc69fa0ec1fe050d